pydm/data_plugins/__init__.py

```diff
@@ -9,14 +9,45 @@
 import logging
 import imp
 import uuid
+try:
+    from Queue import Queue
+except ImportError:
+    from queue import Queue
```

Comment:
we could use instead `collections.deque`:
`
collections.deque is an alternative implementation of unbounded queues with fast atomic append() and popleft() operations that do not require locking.
`